IPs/RSKIP0.md

```diff
@@ -1,3 +1,15 @@
+---
+rskip: 0
+title: RSKIP Purpose and Guidelines
+description: A RSKIP is a design document providing information to the RSK community describing a new feature for RSK, its processes, or its environment.
+status: Purpose and Guidelines
+purpose: Sec
+author: JL (@julianlen)
+layer: Misc
```

Comment:
Misc is not an option, at least I cannot see it in the list. Check "index.html", "Layer" section.

Reply:
Hey, I left this as is for now, but I'm checking with Diego Masini if we'd need to update the "Layer" of this IP or update the list of "Layers" as soon as I know how we want to resolve this I will update it.